presto-hive/src/main/java/com/facebook/presto/hive/HiveStagingFileCommitter.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.hive.HdfsEnvironment.HdfsContext;
+import com.facebook.presto.hive.PartitionUpdate.StagingFileInfo;
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.PrestoException;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Future;
+
+import static com.facebook.presto.hive.HiveErrorCode.HIVE_FILESYSTEM_ERROR;
+import static io.airlift.concurrent.MoreFutures.getFutureValue;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class HiveStagingFileCommitter
+        implements StagingFileCommitter
+{
+    private final HdfsEnvironment hdfsEnvironment;
+    private final Executor renameExecutor;
```

Comment:
This is called `renameExecution` in `HiveMetadataFactory`, please update it to be `renameExecutor` too.